packages/jest-worker/src/types.ts

```diff
@@ -130,26 +133,26 @@ export type ChildMessage =
 
 export type ParentMessageOk = [
   typeof PARENT_MESSAGE_OK, // type
-  any, // result
+  unknown // result
 ];
 
 export type ParentMessageError = [
   PARENT_MESSAGE_ERROR, // type
   string, // constructor
   string, // message
   string, // stack
-  any, // extra
+  unknown // extra
 ];
 
 export type ParentMessage = ParentMessageOk | ParentMessageError;
 
 // Queue types.
-export type OnStart = WorkerInterface => void;
-export type OnEnd = (?Error, ?any) => void;
-
-export type QueueChildMessage = {|
-  request: ChildMessage,
-  onStart: OnStart,
-  onEnd: OnEnd,
-  next?: QueueChildMessage,
-|};
+export type OnStart = (worker: WorkerInterface) => void;
+export type OnEnd = (err: Error | null, result: unknown | undefined) => void;
```

Comment:
`unknown` includes `undefined`
```suggestion
export type OnEnd = (err: Error | null, result: unknown) => void;
```